src/vm/crst.h

```diff
@@ -106,6 +106,14 @@ extern DWORD g_fEEShutDown;
 extern Volatile<LONG> g_ShutdownCrstUsageCount;
 extern Volatile<LONG> g_fForbidEnterEE;
 
+#ifndef NOINLINE
```

Comment:
We should define it once e.g. somewhere around here: https://github.com/dotnet/coreclr/blob/master/src/inc/palclr.h#L468

Reply:
done. Agree it is much more cleaner now.